.github/workflows/early-access.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         id: vars
         shell: bash
         run: |
-          echo VERSION::$(cat VERSION) >>$GITHUB_OUTPUT
+          echo VERSION=$(cat VERSION) >>$GITHUB_OUTPUT
```

Comment:
Don't forget to surround the echoed expressions with quotes, such as
`echo "VERSION=$(cat VERSION)" >>$GITHUB_OUTPUT`

Reply:
Ah yes, the PR still in draft state commented out few code to make the workflow run in my fork repo, looks like they pass without quotes too, still will add them